sender.go

```diff
@@ -342,7 +343,12 @@ func (s *LineSender) Long256Column(name, val string) *LineSender {
 		return s
 	}
 	s.buf.WriteByte('=')
-	s.lastErr = s.writeStrValue(val, false)
+	hexVal := "0x"
+	if utf8.RuneCountInString(val.Text(16))%2==1 {
+		hexVal += "0"
+	}
+	hexVal += val.Text(16) + "i"
```

Comment:
`val.Text(16)` is an expensive call. [`Append`](https://pkg.go.dev/math/big#Int.Append) should be used instead. To do that, you should add a `func (b *buffer) WriteBigInt(i *big.Int)` method similar to `WriteInt` and `WriteFloat`.